python/CMakeLists.txt

```
@@ -31,6 +31,7 @@ project(
 
 option(FIND_RMM_CPP "Search for existing RMM C++ installations before defaulting to local files"
        OFF)
+option(RMM_BUILD_WHEELS "Whether we're building a wheel for pypi" OFF)
```

Two bits:
- avoid “we”
- the wheels don’t currently go to PyPI
```suggestion
option(RMM_BUILD_WHEELS "Whether this build is generating a Python wheel" OFF)
```